dbms/src/Formats/ProtobufReader.cpp

```diff
@@ -67,7 +68,10 @@ bool ProtobufReader::SimpleReader::startMessage()
         if (unlikely(in.eof()))
             return false;
         size_t size_of_message = readVarint();
+        if (size_of_message == 0)
+            throwUnknownFormat();
```

Comment:
I don't think we should throw an exception in this case.
A message with all field set by default doesn't seem to be incorrect.

Reply:
Fixed in https://github.com/yandex/ClickHouse/pull/6132.